script/review-logs.ts

```diff
@@ -0,0 +1,17 @@
+/* tslint:disable:no-sync-functions */
+
+import * as fs from 'fs'
+import * as path from 'path'
+const { getUserDataPath } = require('./dist-info')
```

Comment:
Could we give `getUserDataPath` an explicit type annotation? Presumably it's being imported as `any`.